make/Makefile.container.mk

```diff
@@ -44,7 +44,14 @@ container-build-operator: .ensure-operator-sdk-exists
 	cd "${ROOTDIR}/operator" && "${OP_SDK}" build --image-builder ${DORP} --image-build-args "--pull" "${OPERATOR_QUAY_TAG}"
 
 ## container-build: Build Kiali and Kiali operator container images
-container-build: container-build-kiali container-build-operator
+container-build:
+# On x86_64 machine, build both kiali and operator images.
+ifeq ($(GOARCH),amd64)
+	make container-build-kiali container-build-operator
+# On other achitectures, only build kiali image.
+else
+	make container-build-kiali
+endif
```

Comment:
You will need a similar conditional for the `container-push` target, to avoid pushing the operator image that won't be built if `GOARCH` is not amd64.

Reply:
Done. I will check how to make the operator work and the process of that feature release. If they support arm64, we can also remove both conditionals.